dashboard/app/views/projects/index.html.haml

```diff
@@ -20,6 +20,8 @@
 
   #new-project-buttons
 
+  #my-projects
```

Comment:
we've been trying to mimic the angular style for these ids so it's super clear where we are adding react. I think something like `#react-my-projects` would be a bit more clear.